exporting/pubsub/pubsub.c

```diff
@@ -69,13 +98,16 @@ void pubsub_connector_worker(void *instance_p)
     struct pubsub_specific_config *connector_specific_config = instance->config.connector_specific_config;
     struct pubsub_specific_data *connector_specific_data = instance->connector_specific_data;
 
-    while (!netdata_exit) {
+    while (!instance->engine->exit) {
         struct stats *stats = &instance->stats;
         char error_message[ERROR_LINE_MAX + 1] = "";
 
         uv_mutex_lock(&instance->mutex);
         uv_cond_wait(&instance->cond_var, &instance->mutex);
 
+        if (unlikely(instance->engine->exit))
```

Comment:
I think that when we are exiting here, we also need to unlock the mutex, I already changed this in other PR.

Reply:
Fixed.